client/a8c-for-agencies/components/sidebar-menu/hooks/use-main-menu-items.tsx

```diff
@@ -116,6 +117,19 @@ const useMainMenuItems = ( path: string ) => {
 			},
 			...referralItems,
 			migrationMenuItem,
+			...( isSectionNameEnabled( 'a8c-for-agencies-woopayments' )
+				? [
+						{
+							sidebarIcon: <img src={ wooPaymentsIcon } alt="WooPayments" />,
```

Comment:
I believe we don't need to create another prop. We can use the existing icon prop instead.

```
							icon: <img src={ wooPaymentsIcon } alt="WooPayments" />,
```